utilities/src/main/java/io/syndesis/qe/wait/OpenShiftWaitUtils.java

```diff
@@ -222,4 +235,13 @@ public static void assertEventually(String message, BooleanSupplier condition, l
     public static void assertEventually(String message, BooleanSupplier condition) throws InterruptedException {
         assertEventually(message, condition, DEFAULT_WAIT_INTERVAL, 5 * 60 * 1000);
     }
+
+    public static void waitForPodIsReloaded(String podName) throws InterruptedException, TimeoutException {
+
+        //this process o pod reloading:
+        //1. after extension have been uploaded there is short time period (<10sec) when deployment of new(reloaded) pod haven't started yet.
+        Thread.sleep(10000);
```

Comment:
Please use `TestUtils.sleepIgnoreInterrupt()`, we want to minimize number of thrown exceptions in our util module methods, otherwise we have to use try-catch blocks every time we use those methods.